src/mock-builders/translator.js

```diff
@@ -3,3 +3,9 @@ export const mockTranslatorFunction = (value, params = {}) =>
     const regexp = new RegExp(`\\{\\{\\s${key}\\s\\}\\}`, 'g');
     return acc.replace(regexp, params[key]);
   }, value);
+
+export const mockTranslationContext = {
+  // Mock translation function that always falls back to the key.
+  // It also handles nested keys, e.g. 'aria/Send' will fall back to 'Send'.
+  t: (key) => key.split('/').pop(),
```

Comment:
We don't load translations in tests, so all translations fall back to the key. However, we don't want translations like `t('aria/Send')` to fall back to `'Send'` and not ~`'aria/Send'`~.

For test cases where that's an issue, `mockTranslationContext` can be provided.